app/src/sharedTest/java/org/oppia/android/app/parser/ListItemLeadingMarginSpanTest.kt

```diff
@@ -485,11 +487,14 @@ class ListItemLeadingMarginSpanTest {
   @RunOn(TestPlatform.ROBOLECTRIC, buildEnvironments = [BuildEnvironment.BAZEL])
   fun testDrawLeadingMargin_forNestedBulletItems_inRtl_isDrawnCorrectlyWithIndentation() {
     val displayLocale = createDisplayLocaleImpl(EGYPT_ARABIC_CONTEXT)
+    val width: Int = context.resources.displayMetrics.widthPixels
 
     val canvas = Canvas()
+    shadowOf(canvas).setWidth(width)
```

Comment:
We should also be setting up the LTR tests to have a correct canvas, too. Presumably it won't change the outcomes, but consistency is important here since, without setting width, the canvas isn't being set up well for verifying what these tests are trying to verify.

Reply:
Done.